### ISSUE TITLE
New Release

### ISSUE THREAD

**Issue by anwarchk** [Azure/NoOpsAccelerator#231]
Can we please cut a new release if that's possible ?

**Comment by sstjean** [comment]
Yes! Yes, we can.  We are reworking the Terraform side of the project and will have that solid in a day or two. January Release to follow.

**Comment by anwarchk** [comment]
Thanks @sstjean
